### PR TITLE
Context Options

### PR DESCRIPTION
**Pull request by redwoodfavorite**
Pipeline for passing options into a rendering context.

### CODE REVIEW

File: core/FamousEngine.js [hunk 1]
```diff
@@ -351,9 +351,13 @@ FamousEngine.prototype.message = function message (command) {
  *
  * @return {Scene} a new instance of Scene.
  */
-FamousEngine.prototype.createScene = function createScene (selector) {
+FamousEngine.prototype.createScene = function createScene (selector, options) {
     selector = selector || 'body';
 
+    this.message('CONTEXT');
```
Comment (alexanderGugel) [inline core/FamousEngine.js]: This would mean we can get rid of `getOrSetContext` and replace it with a simple getter.

Comment (redwoodfavorite) [inline core/FamousEngine.js]: :+1:  Yeah I did remove that function actually.  Changed to 'createContext'.

Comment (alexanderGugel) [inline core/FamousEngine.js]: :+1:

**Comment by michaelobriena** [comment]
@redwoodfavorite can you update this to work with the COmmands scheme?

**Comment by michaelobriena** [comment]
Closing in favor of more explicit options vs using the command buffer.

**Pull request closed without merging**